scripts/install.gs

```diff
@@ -18,6 +18,16 @@ if *goGet {
 	Printfln("go get -u -v %s/%s", GCSE, a)
 	MustSucc(Bash("go get -u -v %s/%s", GCSE, a))
   }
+
+  additionalPackages := []string{
```

Comment:
why this is necessary? Won't previous go get automatically get these dependencies?